python/modules/IcePy/Communicator.cpp

```diff
@@ -259,19 +168,6 @@ communicatorInit(CommunicatorObject* self, PyObject* args, PyObject* /*kwds*/)
         return -1;
     }
 
-    //
-    // Remaining command line options are passed to the communicator
-    // as an argument vector in case they contain plug-in properties.
-    //
-    int argc = static_cast<int>(seq.size());
```

Comment:
Unless I’m missing something, this seems completely unnecessary. We’re calling the C++ `Ice::initialize`, which can take a `StringSeq` by reference and update it.